bin/update-versions.sh

```diff
@@ -19,24 +19,30 @@ fi
 
 # Update the plugin versions
 # merpress.php
-sed -i "s/Version: .*/Version: $plugin/" merpress.php
-sed -Ei "s/(define\( 'MERMAID_PLUGIN_VERSION', ')$version_pattern(.*)/\1$plugin\3/" merpress.php
+
+sed_option=""
+if [ "$(uname)" == "Darwin" ]; then
+    # Do something under Mac OS X platform        
+    sed_option="'' -e"
+fi
+sed -i "${sed_macOS}" "s/Version: .*/Version: $plugin/" merpress.php
+sed -Ei "${sed_macOS}" "s/(define\( 'MERMAID_PLUGIN_VERSION', ')$version_pattern(.*)/\1$plugin\3/" merpress.php
```

Comment:
All these can be changed then too.
```suggestion
sed_i_wrapper -i "s/Version: .*/Version: $plugin/" merpress.php
sed_i_wrapper -i -E "s/(define\( 'MERMAID_PLUGIN_VERSION', ')$version_pattern(.*)/\1$plugin\3/" merpress.php
```